packages/components/src/components/atoms/BaseCard/BaseCard.tsx

```diff
@@ -26,155 +44,79 @@ const useStyles = createUseStyles({
     transition: '0.6s',
   },
   // Style classes
-  card: ({ width, height, cardPosition }) => ({
-    width: width || 600,
-    height: height || 400,
-    perspective: 1000,
-    outline: 'none',
+  card: ({ width, height }) => ({
+    width,
+    height,
+    position: 'absolute',
     overflow: 'hidden',
-    transform: `translate(${cardPosition.x}px, ${cardPosition.y}px)`,
   }),
-  flipped: { transform: 'rotateY(180deg)' },
-  cardFlipper: {
+  flipped: ({ faceUp }) => ({ transform: `rotateY(${faceUp ? 0 : 180}deg)` }),
+  cardFlipper: ({ width, height }) => ({
```

Comment:
I was wondering if we shouldn't create a type to better conduct those styling parameters. So at least we are coherent through the whole file.

```
interface BaseCardThemingOptions {
     width: number;
     height: number;
     faceUp: boolean;
}


// In the styling
cardFlipper: ({width, height}: BaseCardThemingOptions) => {}

```